example_configs/penpot.md

```diff
@@ -0,0 +1,27 @@
+# Penpot Configuration
+
+Penpot is the only design & prototype platform that is deployment agnostic. You can use it or deploy it anywhere.
+
+## LDAP Config
+
+Penpot comes with support for Lightweight Directory Access Protocol.
+
+[Penpot documentation](https://help.penpot.app/technical-guide/configuration/#ldap)
+
+example of configuration:
+
+```yaml
+PENPOT_LDAP_HOST: lldap.shared.svc.cluster.local
+PENPOT_LDAP_PORT: 3890
+PENPOT_LDAP_SSL: false
```

Comment:
Can you add a comment to switch this one to true if using LDAPS? So people are not confused by starttls